extensions/attention-stream/src/youtube/utils/validate.ts

```diff
@@ -0,0 +1,7 @@
+const validate = ({ body, url, hostUrl }) => {
+  if (typeof body === 'object' && typeof url === 'string' && typeof hostUrl === 'string') return;
+
+  throw new Error(`Non-Critical Error - Invalid Data: ${{ body, url, hostUrl }}`);
+};
+
+export default validate;
```

Comment:
Suggestion: Better to export functions as I suggested rather than default exports or const exports. That way imports look way better.